test/Npgsql.Tests/TransactionTests.cs

```diff
@@ -162,35 +162,29 @@ public void EmptyRollback()
         }
 
         [Test, Description("Tests that the isolation levels are properly supported")]
-        public void IsolationLevels()
+        [TestCase(IsolationLevel.ReadCommitted,   "read committed")]
+        [TestCase(IsolationLevel.ReadUncommitted, "read uncommitted")]  // Not actually supported in PostgreSQL
```

Comment:
Even if it's not supported, it doesn't affect the test. So the is no reason to have the comment.

Reply:
That's true, this is just for people browsing test code as documentation...